testutils/rand/rand.go

```diff
@@ -224,7 +224,7 @@ type StringGen struct {
 
 // Strings returns a random string generator that produces strings from the default alphabet of random length in the given limits (upper limit exclusive)
 func Strings(minLength int, shorterThan int) StringGen {
-	alphabet := []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-.:")
+	alphabet := []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-.+-#[]<>")
```

Comment:
why this change?

Reply:
`:` is not a valid character for ibc paths. I'm keeping it so that `StrBetween` is valid for our usage.